### ISSUE TITLE
German Audio Translation, source Bubenheim

### ISSUE THREAD

**Issue by jobbb2** [AlfaazPlus/QuranApp#454]
### Describe your suggested feature

Thank you very much for you hard work !! may allah reward you abundantly!

Please can you add the German Audio Translation, source is from Frank Bubenheim - Publisher King Fahd Complex, from this website:
https://www.youtube.com/@deredlequran4897/videos
I prefer this german reciter, his voice is clearer and he recites more slowly and comprehensibly. In the background is a quiet quran recitation in arabic.

But if you can not use it, here is a second german reciter:
https://www.listennotes.com/podcasts/quran-audio-auf-deutsch-nail-ibn-radhan-hT_I8lbmKLP or
https://podcasts.apple.com/us/podcast/quran-audio-auf-deutsch/id1546182097
In the first link there is a download option.

I hope you can use it, for adding the German Audio Translation.

Best regards

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.

**Comment by faisalcodes** [comment]
I'll look into this.

**Comment by faisalcodes** [comment]
The sources included have chapter wise audios. To include it in the app I'll need verse by verse audio. If you find a source please let me know.

**Comment by faisalcodes** [comment]
Can you confirm for me if the german audio on this page is good enough (scroll down to the "Translation audios" section).

https://qurandb.com/#8297cef2-b62a-4171-8554-84057b2839e3

**Comment by jobbb2** [comment]
Hello, i cannot open it. It need a password!

**Comment by jobbb2** [comment]
> Can you confirm for me if the german audio on this page is good enough (scroll down to the "Translation audios" section).
> 
> https://qurandb.com/#8297cef2-b62a-4171-8554-84057b2839e3

Yes, the german audio is good enough. You can use it. Best regards

**Comment by jobbb2** [comment]
Hello.

I sent you a confirmation 7 months ago so that you could integrate this feature.

How long do you estimate it will take until it is integrated?

Best regards

**Comment by faisalcodes** [comment]
A german audio translation has been added. You can select the new audio from the `settings` under `Select Reciter` section. If you don't see a german audio option, try to refresh the page.

**Comment by jobbb2** [comment]
Nice, Thank you very much
